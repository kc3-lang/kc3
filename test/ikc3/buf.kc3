quote require Buf
require Buf
quote %Buf{}
%Buf{}
