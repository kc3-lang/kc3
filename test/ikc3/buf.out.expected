%Buf{}
%Buf{column: (Sw) 0, flush: (Ptr) 0x0, free: false, line: (Sw) 0, ptr: (Ptr) 0x0, read_only: false, refill: (Ptr) 0x0, rpos: (Uw) 0, save: (Ptr) 0x0, seek: (Ptr) 0x0, size: (Uw) 0, user_ptr: (Ptr) 0x0, wpos: (Uw) 0}
